target/linux/ipq806x/files-4.19/arch/arm/boot/dts/qcom-ipq8064.dtsi

```diff
@@ -633,7 +633,7 @@
 				mux {
```

Comment:
Try to line wrap your commit descriptions to < 72 chars. Otherwise I would need to touch this commit again and line wrap that commit description, thanks.